### PR TITLE
Fix variable.scss path in change_color.sh

### PR DESCRIPTION
**Pull request by ceyhunnabiyev**

### CODE REVIEW

File: change_color.sh [hunk 1]
```diff
@@ -229,7 +229,7 @@ done
       #-e 's/%INACTIVE_TXT_FG%/'"$INACTIVE_TXT_FG"'/g' \
       #-e 's/%INACTIVE_MENU_FG%/'"$INACTIVE_MENU_FG"'/g' \
 
-sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' ./src/_sass/gtk/_variables.scss
+sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' ./src/_sass/_variables.scss
```
Comment (actionless) [inline change_color.sh]:
```
for FILEPATH in ./src/_sass/*/_variables.scss ; do
    sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' "${FILEPATH}"
done
```

Comment (ceyhunnabiyev) [reply change_color.sh]: @actionless , `$material_radius` is not specified in `./src/_sass/gnome-shell/_variables.scss`

**Comment by nana-4** [comment]
Oops, I forgot to update this as well :P
(I was going to recheck it when the ongoing redesign work is done.)

Thanks for the PR!

File: change_color.sh (hunk 1 above)
Comment (nana-4) [reply change_color.sh]: @actionless Don't worry, I won't add `$material_radius` into `./src/_sass/gnome-shell/_variables.scss` :smile:

**Pull request merged**